lib/GameConstants.h

```diff
@@ -745,6 +745,27 @@ DLL_LINKAGE std::ostream & operator<<(std::ostream & os, const ETerrainType acti
 
 ID_LIKE_OPERATORS_DECLS(ETerrainType, ETerrainType::EETerrainType)
 
+class DLL_LINKAGE EPathfindingLayer
+{
+public:
+	enum EEPathfindingLayer
+	{
+		WRONG = -2, AUTO = -1, LAND = 0, SAIL = 1, WATER, AIR, NUM_LAYERS
+	};
+
+	EPathfindingLayer(EEPathfindingLayer _num = WRONG) : num(_num)
+	{}
+
+	ID_LIKE_CLASS_COMMON(EPathfindingLayer, EEPathfindingLayer)

```

Comment:
The reason is to define common operators (GameConstats.h, line 96), but also default constructor and more I didn't spot immediatelly. Whether they are neccessary or not remains a question.